libs/npm/core/src/components/formik/sentiment/FormikSentiment.tsx

```diff
@@ -92,11 +113,11 @@ export const FormikSentiment = <T extends object>({
               key={option}
               onClick={() => {
                 setActive(option);
-                setFieldValue(
-                  name.toString(),
-                  !!defaultTonePool ? [{ ...defaultTonePool, value: option }] : [],
-                  true,
-                );
+                const updatedTonePools = !!defaultTonePool
+                  ? [{ ...defaultTonePool, value: option }]
+                  : [];
+                setFieldValue(name.toString(), updatedTonePools, true);
```

Comment:
This is going to modify the the formik context field and also call the `onSentimentChange`.  You can pass the valid name to the component to control what is being updated.  Or you can choose to only perform the `onSentimentChange` if it has been provided.  Doing both is a bad component.